packages/nextlove/package.json

```diff
@@ -44,6 +45,7 @@
     "yalc": "npm run build && yalc push"
   },
   "peerDependencies": {
+    "esbuild": "^0.14.7",
```

Comment:
esbuild was already implicitly a peer dependency because of `esbuild-register` (not exactly sure why that's a peer in the first place...)